public/coding/JavaScript/JsFunctions/runningPaceCalculator/runningPaceCalculator-Practice.js

```diff
@@ -0,0 +1,22 @@
+function runningPaceCalculator(distanceMiles, timeHours, timeMinutes) {
+  /*
+   * In this challenge you will calculate the running pace per mile given the following parameters:
+   * 1) the race distance in miles, 2) the time in hours, and 3) time in minutes to complete the race
+   * The pace per mile should be out put in the HH:MM format. If a 13.1 mile race takes 1:45 to complete, the pace per mile function should output 8:35
```

Comment:
I think this function should just take in the time in minutes. My students are just learning about functions, and we can simplify this problem for them by only having to worry about two function parameters. 

In your solution you can rid of the calculation for `totalTimeMinutes` and just assume that is what is passed in for the minutes paramter.